internal/stores/objects.go

```diff
@@ -327,21 +327,21 @@ type (
 		ID     uint64 `gorm:"primaryKey"`
 		SlabID uint64 `gorm:"index;NOT NULL"`
 
-		SectorID []byte `gorm:"index;NOT NULL"` // No CASCADE to not delete sectors
+		SectorID consensus.Hash256 `gorm:"index;NOT NULL;type:bytes;serializer:gob"` // No CASCADE to not delete sectors
```

Comment:
`gob`? 😱
does gorm support `encoding.TextMarshaler`?

Reply:
nvm, I see we're kinda forced to use it in other places anyway (e.g. for `CoveredFields`). oh well. I just have some lingering bias against `gob` because it's not idempotent.

Reply:
haha if we feel super motivated we can implement a custom serializer but I thought that might be a bit much for now